### DISCUSSION TITLE
Apoio na ligação ao API da TP-Link

### DISCUSSION THREAD

**Discussion by jmmagomes** [tnmendes/watt#77] [General]
Olá Tiago,
Vi a tua aplicação para iOS de controlo dos dispositivos Tapo, da TP-Link.
O que eu pretendo é fazer a ligação através dos API para registar os consumos. Já consegui, vendo uma aplicação para fazer a essa ligação (https://flows.nodered.org/node/node-red-contrib-tplink-tapo-connect-api), ver os dispositivos que tenho associados.
No entanto, não consigo perceber como fazer a ligação a cada dispositivo para recolher os dados de consumo.
Gostaria de saber se me podes ajudar nisto.
Obrigado
Jose Mario Gomes

**Comment by tnmendes** [comment]
Olá @jmmagomes só agora é que vi a tua mensagem. Existem varios projectos no github que podes usar para entender com funcionar API da tp-link por exemplo tens este aqui:
https://github.com/adumont/tplink-cloud-api

Boa sorte com o teu projecto
